internal/connect/command_custom_plugin_create.go

```diff
@@ -39,10 +39,10 @@ func (c *customPluginCommand) newCreateCommand() *cobra.Command {
 	cmd.Flags().String("plugin-file", "", "ZIP/JAR custom plugin file.")
 	cmd.Flags().String("connector-class", "", "Connector class of custom plugin.")
 	cmd.Flags().String("connector-type", "", "Connector type of custom plugin.")
-	cmd.Flags().String("cloud", "", "Cloud provider of custom plugin. Supported value [\"aws\",\"azure\"] (default \"aws\").")
 	cmd.Flags().String("description", "", "Description of custom plugin.")
 	cmd.Flags().String("documentation-link", "", "Document link of custom plugin.")
 	cmd.Flags().StringSlice("sensitive-properties", nil, "A comma-separated list of sensitive property names.")
+	pcmd.AddPluginCloudFlag(cmd)
```

Comment:
Will this flag be used for connect and flink?

Reply:
We do not support flink artifact cloud attribute now. 
This is more like a temporary flag, cuz we will also support gcp in the future.

Reply:
Let's define this function in the current package then. No need to define it in `pkg/cmd` if it's not going to be used globally.